tests/qos/files/cisco/qos_param_generator.py

```diff
@@ -82,12 +82,12 @@ def __init__(self, qos_params, duthost, dutAsic, topo, bufferConfig, portSpeedCa
             self.log("Pre-pad drop thr bytes:    {}".format(pre_pad_drop))
             self.log("Drop thr bytes:            {}".format(self.drop_thr))
             self.log("Reduced pause thr bytes:   {}".format(self.reduced_pause_thr))
-        self.config_facts = duthost.get_running_config_facts()
-        # DSCP value for lossy
-        self.dscp_queue0 = self.get_one_dscp_from_queue(0)
-        self.dscp_queue1 = self.get_one_dscp_from_queue(1)
-        # DSCP, queue, weight list
-        self.dscp_list, self.q_list, self.weight_list = self.get_dscp_q_weight_list()
+            self.config_facts = duthost.asics[0].config_facts(source="running")["ansible_facts"]
```

Comment:
instead of asics[0], can we use duthost.asic_instance().config_facts(xxx) ?

Reply:
Updated to use asic_instance().